test/integration/token_comment_test.rb

```diff
@@ -32,5 +32,21 @@ class TokenCommentTest < ActionDispatch::IntegrationTest
     assert_equal User.find(comment.uid).email, "jeff@publiclab.org"
     f.close()
   end
+   test 'should parse incoming mail from yahoo service correctly and add comment' do
+    require 'mail'
+    mail = Mail.read('test/fixtures/incoming_test_emails/yahoo/incoming_yahoo_email.eml')
+    node = Node.last
+    mail.subject = "Re: #{node.title} (##{node.nid})"
+    Comment.receive_mail(mail)
+    f = File.open('test/fixtures/incoming_test_emails/yahoo/final_parsed_comment.txt', 'r')
+    comment = Comment.last
+    user_email = mail.from.first
+    assert_equal comment.comment, f.read
+    assert_equal comment.nid, node.id
+    assert_equal comment.message_id, mail.message_id
+    assert_equal comment.comment_via, 1
+    assert_equal User.find(comment.uid).email, user_email
```

Comment:
Move this test by creating a new file in `incoming_test_parsing_test` folder like done for Gmail - https://github.com/publiclab/plots2/tree/master/test/integration/incoming_mail_parsing_test

Reply:
> Move this test by creating a new file in `incoming_test_parsing_test` folder like done for Gmail - https://github.com/publiclab/plots2/tree/master/test/integration/incoming_mail_parsing_test

So I have to remove the test from here or it is meant to be there for both locations?